### DISCUSSION TITLE
Waiver Priority

### DISCUSSION THREAD

**Discussion by keithtruong** [cwendt94/espn-api#574] [ESPN API] [answered]
Hi all, thanks for this -- it's been super helpful!

I'm looking to keep track of waiver priority per team in my fantasy football league. Is this information available through the API? I wasn't able to find it. Thank you!

**Comment by cwendt94** [comment, marked as answer]
It is currently not grabbed but should be an easy add to the Team class. I already checked that data is returned by ESPN's API!

**Reply by cwendt94** [reply]
This is now in v0.39.0! In the Team class var `waiver_rank `

**Reply by keithtruong** [reply]
Works like a charm! Thanks a ton, Christian!
